### ISSUE TITLE
Optimierung Google Suchergebnisse der einzelnen Extensions (extensions.contao.org)

### ISSUE THREAD

**Issue by contaoacademy** [contao/package-list#2]
Die Suchergebnisse bei der Suche nach bestimmten Erweiterungen passen nicht zusammen.
Evtl. kann das optimiert werden.

Testen mit
`site:extensions.contao.org`

Aktueller Stand siehe Screenshot:
![suche-extensions](https://user-images.githubusercontent.com/23719181/72803331-1ffcd380-3c4e-11ea-80de-1c847ff94ea6.jpg)

**Comment by aschempp** [comment]
Ich habe die Liste angepasst, sodass Seitentitel und Page Description im Popup aktualisiert werden. Mal sehen welche Auswirkungen das hat …

**Comment by leofeyer** [comment]
@contaoacademy Kannst Du nochmal prüfen, ob @aschempp's Änderungen etwas gebracht haben?

**Comment by contaoacademy** [comment]
Ich kann bisher leider keine Verbesserung bei der Beschreibung feststellen.
Es sieht so aus, als ob die Daten von den promoted Extensions sehr häufig als Beschreibung verwendet werden.

![site:extensions contao org - Google-Suche 2020-02-19 20-41-13](https://user-images.githubusercontent.com/23719181/74869426-39319680-5358-11ea-80e7-0fcafc7cb372.jpg)

**Comment by leofeyer** [comment]
Ich nehme an, dass Google das Modal-Window verwirrt und die Beschreibung aus dem normalen Fenster kommt.

**Comment by aschempp** [comment]
Das scheint mir auch so. Ich habe eingebaut dass das Modal den Seitentitel und die Meta-Beschreibung entsprechend aktualisiert, aber das scheint Google (noch) nicht verstanden zu haben.

**Comment by contaoacademy** [comment]
Was mir noch aufgefallen ist, dass in der deutschen Google Suche die englischen Texte stehen.

**Comment by aschempp** [comment]
Ja, auch das scheint Google leider nicht zu verstehen. Ich hab mich mal kurz damit beschäftigt, und die SEO-Experten sagen dass Google zwar behautet Javascript zu "verstehen", aber die einzig wahr Lösung wohl doch Server-Side-Rendering ist. Und das können wir (aktuell) nicht 😢

**Comment by contaoacademy** [comment]
Verstehe. Sollen wir damit das Ticket schließen oder noch offen lassen?

**Comment by aschempp** [comment]
Gerne offen lassen, ich schaue von Zeit zu Zeit ob ich daran was ändern kann.

**Comment by lukasbableck** [comment]
`data-nosnippet` on `div.layout-main` could help in this case. We should also set `inert`, as @ausi suggested on Slack. Although this might not be interpreted by Google.

**Comment by ausi** [comment]
Just for clarification, `data-nosnippet` and `inert` should only be present as long as the modal is open.

**Comment by aschempp** [comment]
Improved in 4039626c923fa0fa62d2e64ffec9713b92abb558

**Comment by aschempp** [comment]
has there been any improvements in the search results?

**Comment by zoglo** [comment]
Looks like it

![Image](https://github.com/user-attachments/assets/0bf4c3b8-b471-4a09-9387-4031525c9285)

**Comment by leofeyer** [comment]
Some entries are still wrong though:

<img width="702" alt="" src="https://github.com/user-attachments/assets/6d7d0ac3-f31e-4368-aff1-dc5322815295" />

**Comment by fritzmg** [comment]
It will probably take time until all entries are corrected in the index.

**Comment by aschempp** [comment]
interesting, why does the extension list find `symfony/finder` etc?

**Comment by zoglo** [comment]
> interesting, why does the extension list find `symfony/finder` etc?

It crawls through the "requirements" of packages as well
